install-deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
how's your test going? i see you instruct user to create a symlink in README.FreeBSD, so we can dispense with this change?